# --- T2-COPYRIGHT-NOTE-BEGIN ---
# This copyright note is auto-generated by ./scripts/Create-CopyPatch.
# 
# T2 SDE: package/.../ebtables/iets.patch
# Copyright (C) 2004 - 2005 The T2 SDE Project
# 
# More information can be found in the files COPYING and README.
# 
# This patch file is dual-licensed. It is available under the license the
# patched project is licensed under, as long as it is an OpenSource license
# as defined at http://www.opensource.org/ (e.g. BSD, X11) or under the terms
# of the GNU General Public License as published by the Free Software
# Foundation; either version 2 of the License, or (at your option) any later
# version.
# --- T2-COPYRIGHT-NOTE-END ---
--- ebtables-v2.0.6/ebtables.c.old	Thu Sep  9 22:39:49 2004
+++ ebtables-v2.0.6/ebtables.c	Thu Sep  9 22:40:18 2004
@@ -1170,7 +1170,7 @@ static int check_rule_exists(int rule_nr
 		if (!t->compare(t->t, u_e->t))
 			continue;
 		return i;
-letscontinue:
+letscontinue:;
 	}
 	return -1;
 }
